src/streambench/gearpumpbench/src/main/scala/com/intel/hibench/streambench/gearpump/RunBench.scala

```diff
@@ -55,14 +55,14 @@ object RunBench {
   }
 
   private def getConfig(conf: ConfigLoader): GearpumpConfig = {
-    val benchName = conf.getProperty("hibench.streambench.benchname")
-    val topic = conf.getProperty("hibench.streambench.topic_name")
-    val zkHost = conf.getProperty("hibench.streambench.zookeeper.host")
-    val consumerGroup = conf.getProperty("hibench.streambench.consumer_group")
+    val benchName = conf.getProperty(StreamBenchConfig.TESTCASE)
+    val topic = conf.getProperty(StreamBenchConfig.KAFKA_TOPIC)
+    val zkHost = conf.getProperty(StreamBenchConfig.ZK_HOST)
+    val consumerGroup = conf.getProperty(StreamBenchConfig.CONSUMER_GROUP)
     val kafkaPartitions = conf.getProperty("hibench.streambench.partitions").toInt
```

Comment:
I would prefer have an centralized place to map configuration, it would be easy to maintain. For storm specific config, we can recognized by name, STORM_XXXX, just like SPARK_XXXX, FLINK_XXXX.